docs/api.md

```diff
@@ -0,0 +1,376 @@
+# API
+
+## Surface
+
+Container for all other ART components
+
+|   Prop   |     Type     | Default |
+| :------: | :----------: | :-----: |
+|  height  |   `number`   |   `0`   |
+|  width   |   `number`   |   `0`   |
+|  style   |    `any`     |   ---   |
+| children | `React.Node` |   ---   |
+
+```js
+import {Dimensions} from 'react-native';
+import {Surface} from '@react-native-community/art';
+
+const SURFACE_DIMENSIONS = Dimensions.get('window').width;
```

Comment:
It's not a good practice to call dimensions outside of render, can you move it into the `Heart` function?